app/scripts/controllers/authentication/authentication-controller.ts

```diff
@@ -0,0 +1,247 @@
+import {
```

Comment:
Was there consideration given to adding these controllers as their own package in https://github.com/MetaMask/core?

Reply:
Discussed, we'll keep this within extension & will move out once we start mobile notifications :)